src/main/java/net/sf/jabref/logic/importer/fileformat/RisImporter.java

```diff
@@ -129,7 +129,11 @@ public ParserResult importDatabase(BufferedReader reader) throws IOException {
                         fields.put(FieldName.TITLE, fields.get(FieldName.TITLE).replaceAll("\\s+", " ")); // Normalize whitespaces
                     } else if ("BT".equals(tag)) {
                         fields.put(FieldName.BOOKTITLE, value);
-                    } else if ("T2".equals(tag) || "JO".equals(tag)) {
+                    } else if ("T2".equals(tag)&&(fields.get(FieldName.JOURNAL) == null || "".equals(fields.get(FieldName.JOURNAL)))){
```

Comment:
Could you fix the formatting in the condition here? I am referring to whitespaces between operators `&&` and the operands.